modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// */openshift_pipelines/installing-pipelines.adoc
+[id="op-installing-pipelines-operator-in-web-console_{context}"]
+= Installing the {pipelines-title} Operator in web console
+
+You can install {pipelines-title} using the Operator listed in the {product-title} OperatorHub. When you install the {pipelines-title} Operator, the Custom Resources (CRs) required for the Pipelines configuration are automatically installed along with the Operator.
+
+[discrete]
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to *Operators* -> *OperatorHub*.
+
+. Use the *Filter by keyword* box to search for `{pipelines-title} Operator` in the catalog. Click the *OpenShift Pipelines Operator* tile.
++
+[NOTE]
+====
+Ensure that you do not select the *Community* version of the *OpenShift Pipelines Operator*.
+====
++
+//image::op-installed-tile.png[]
+//Remove the note below once the community operator is removed from OperatorHub
+
+. Read the brief description about the Operator on the *{pipelines-title} Operator* page. Click *Install*.
++
+//image::op-install-subscription.png[]
+
+. On the *Create Operator Subscription* page:
+
+  .. Select *All namespaces on the cluster (default)* for the *Installation Mode*. This mode installs the Operator in the default `openshift-operators` namespace, which enables the Operator to watch and be made available to all namespaces in the cluster.
+
+  .. Select *Automatic* for the *Approval Strategy*. This ensures that the future upgrades to the Operator are handled automatically by the the Operator Lifecycle Manager (OLM). If you select the *Manual* approval strategy, OLM creates an update request. As a cluster administrator, you must then manually approve the OLM update request to update the Operator to the new version.
+  .. Select an *Update Channel*.
+    *** The *ocp-<4-x>* channel enables installation of the latest stable release of the {pipelines-title} Operator.
+    *** The *preview* channel enables installation of the latest preview version of the {pipelines-title} Operator, which may contain features that are not yet available from the 4-x update channel.
```

Comment:
I suggest to unify channel name - replace `4-x` with `ocp-<4-x>`. The actual channel name is `ocp-4.4` so maybe `ocp-<4.x>` would be a better placeholder.

Reply:
done